### PR TITLE
configurable outgoing mailbox

### PR DESCRIPTION
**Pull request by lushenle**
#147
supported to be configurable the outgoing mail

### CODE REVIEW

**Review by tianzhou — APPROVED** [review]
LGTM

File: src/pages/api/auth/[...nextauth].ts [hunk 1]
```diff
@@ -18,7 +18,7 @@ export const authOptions: NextAuthOptions = {
     ? [
         EmailProvider({
           server: process.env.EMAIL_SERVER,
-          from: "noreply@sqlchat.ai",
+          from: process.env.EMAIL_FROM,
```
Comment (tianzhou) [inline src/pages/api/auth/[...nextauth].ts:21]: Please include "noreply@sqlchat.ai" as the default if not specified anything

**Review by tianzhou — CHANGES_REQUESTED** [review]

File: src/pages/api/auth/[...nextauth].ts (hunk 1 above)
Comment (lushenle) [reply src/pages/api/auth/[...nextauth].ts]: > Please include "[noreply@sqlchat.ai](mailto:noreply@sqlchat.ai)" as the default if not specified anything

我的失误，考虑不周了^_^

**Review by tianzhou — APPROVED** [review]
LGTM

**Comment by tianzhou** [comment]
Thanks for the contribution

**Pull request merged**